agent/api/task/task_linux.go

```diff
@@ -272,7 +273,7 @@ func (task *Task) initializeExecCommandAgentResources(cfg *config.Config) error
 		return err
 	}
 
-	execCommandAgentBinNames := []string{execCommandAgentBinName, execCommandAgentSessionWorkerBinName, execCommandAgentSessionLoggerBinName}
+	execCommandAgentBinNames := []string{ExecCommandAgentBinName, ExecCommandAgentSessionWorkerBinName, ExecCommandAgentSessionLoggerBinName}
```

Comment:
The SessionLogger is not necessary for Linux.

Reply:
does the ExecAgent take care of logging?

Reply:
removed